src/test/ui/feature-gates/feature-gate-const_extern_fn.rs

```diff
@@ -0,0 +1,10 @@
+// Check that `const extern fn` and `const unsafe extern fn` is feature-gated
+
+const extern fn foo1() {} //~ ERROR `const extern fn` definitions are unstable
```

Comment:
These should be baked into another item `#[cfg(FALSE)] fn stuff() { ... }` (the point is to test the pre-expansion aspect of gating).